scripts/main.js

```diff
@@ -302,4 +302,53 @@ class Cart {
     clear() {
         this.storage.clear();
     }
+}
+
+/*Here starts engaging with product's add buttons.
+* I synchronized buttons with Cart, and on click
+* i'm parsing html product into object and create
+* instance of ProductModel with its data.
+* Then i'm adding this instance to localStorage via Cart.*/
+const cart = new Cart();
+const bagCounters = document.getElementById("bag");
+const addBtns = document.getElementsByClassName("product__addBtn");
+
+for (let i = 0; i < addBtns.length; i++) {
+    addBtns[i].addEventListener("click", function () {
+        const parent = this.parentNode;
+        const id = parent.dataset.id;
+        const img = parent
+            .getElementsByClassName("product__img")[0]
+            .getElementsByTagName("img")[0]
+            .getAttribute("src");
+        const name = parent
+            .getElementsByClassName("product__description")[0]
```

Comment:
Why did you do this? It is useless call. Do just `parent.getElementsByClassName("product__description__title")[0].innerText`.